lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -90,7 +92,7 @@ sub call_rpc {
     my $before_get_rpc_response_hook = delete($req_storage->{before_get_rpc_response}) || [];
     my $after_got_rpc_response_hook  = delete($req_storage->{after_got_rpc_response})  || [];
     my $before_call_hook             = delete($req_storage->{before_call})             || [];
-    my $rpc_failure_cb               =  delete($req_storage->{rpc_failure_cb}) || 0;
+    my $rpc_failure_cb = delete($req_storage->{rpc_failure_cb}) // undef;
```

Comment:
The `// undef` here is a no-op: if the key isn't in the hash then `delete` will return `undef`.